functions/Get-DbaMemoryUsage.ps1

```diff
@@ -0,0 +1,137 @@
+
+Default output includes columns Server, counter instance, counter, number of pages, memory in KB, memory in MB
+
+.PARAMETER ComputerName
+The SQL Server that you're connecting to.
+
+.PARAMETER Credential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER Simple
+Shows concise information including Server name, Database name, and the date the last time backups were performed
+
+.NOTES
+Author: Klaas Vandenberghe ( @PowerDBAKlaas )
+
+dbatools PowerShell module (https://dbatools.io)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.
+
+.LINK
+ https://dbatools.io/Get-DbaMemoryUsage
+
+.EXAMPLE
+Get-DbaMemoryUsage -ComputerName ServerA
+
+Returns a custom object displaying Server, counter instance, counter, number of pages, memory in KB, memory in MB
+
+.EXAMPLE
+Get-DbaMemoryUsage -ComputerName ServerA\sql987 -Simple
+
+Returns a custom object with Server, counter instance, counter, number of pages, memory in KB, memory in MB
+
+.EXAMPLE
+Get-DbaMemoryUsage -ComputerName ServerA\sql987 | Out-Gridview
+
+Returns a gridview displaying Server, counter instance, counter, number of pages, memory in KB, memory in MB
+
+#>
+	[CmdletBinding()]
+	Param (
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("Host")]
+		[string[]]$ComputerName,
+		[PsCredential]$Credential,
+		[switch]$Simple
+	)
+
+
+	BEGIN
+	{
+    if ($Simple)
+        {
+        $Memcounters = '(Total Server Memory |Target Server Memory |Connection Memory |Lock Memory |SQL Cache Memory |Optimizer Memory |Granted Workspace Memory |Cursor memory usage|Maximum Workspace)'
+        $Plancounters = 'total\)\\cache pages'
+        $BufManpagecounters = 'Total pages'
+        }
+    else
+        {
+        $Memcounters = '(Total Server Memory |Target Server Memory |Connection Memory |Lock Memory |SQL Cache Memory |Optimizer Memory |Granted Workspace Memory |Cursor memory usage|Maximum Workspace)'
+        $Plancounters = '(cache pages|procedure plan|ad hoc sql plan|prepared SQL Plan)'
+        $BufManpagecounters = '(Free pages|Reserved pages|Stolen pages|Total pages|Database pages|target pages|extension .* pages)'
+        }
+
+    }
+
+	PROCESS
+	{
+        foreach ($servername in $ComputerName)
+        {
+            if ($servername -match '\\')
+			{
+				$servername = $servername.Split('\\')[0]
```

Comment:
They may throw an aliased server name. Please use the newly created internal command Resolve-IpAddress to get the IP. You can throw a few things at it: a string, an smo object or even a string with a named instance.